scrapyrt/resources.py

```diff
@@ -74,9 +75,13 @@ def handle_error(self, exception_or_failure, request):
         return self.format_error_response(exception, request)
 
     def format_error_response(self, exception, request):
+        # Python exceptions don't have message attribute in Python 3+ anymore.
+        # Twisted HTTP Error objects still have 'message' attribute even in 3+
+        # and they fail on str(exception) call.
+        msg = exception.message if hasattr(exception, 'message') else str(exception)
```

Comment:
Twisted HTTP Errors do have exc.args, but message is not always first index, for example sample 400 error can look like this:

```
In [2]: exception
Out[2]: twisted.web.error.Error('404', "'Spider not found: dmozx'", None)

In [3]: exception.args
Out[3]: ('404', "'Spider not found: dmozx'", None)
```
this means we can't rely on exc.args[0] refering to message.